src/applications/gi/reducers/calculator.js

```diff
@@ -69,6 +76,91 @@ export default function (state = INITIAL_STATE, action) {
         ...newState
       };
     }
+    case FETCH_BAH_FAILED: {
+      const { beneficiaryZipFetched } = action;
+      const { error } = action.payload;
+
+      // response mismatch - do nothing
+      if (beneficiaryZipFetched !== state.beneficiaryZipFetched) {
+        return state;
+      }
+
+      const newState = {
+        beneficiaryZipError: error,
```

Comment:
Response error gets displayed as an input error. Not sure if this is idiomatic i.e. if response errors should be displayed on a different label.

Reply:
That seems reasonable to me. It's an error with the zip you entered (or trying to use it), so it seems like it should be an input error.